tests/workflow_tests/test_deployment_update.py

```diff
@@ -2004,3 +2012,104 @@ def test_remove_deployment(self):
         self.assertEquals(len(deployment_update_list), len(undel_depups_ids))
         for i in deployment_update_list.items:
             self.assertIn(i['id'], undel_depups_ids)
+
+
+class TestDeploymentUpdateMisc(DeploymentUpdateBase):
```

Comment:
I figured mixed operations are about mixing of specific updates, e.g. node and relationship update; this class is about misc update scenarios. do you still think they should be unified?